tests/plugins/clipboard/manual/draganddropimageintable.html

```diff
@@ -0,0 +1,24 @@
+<textarea id="editor" >
+	<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+		<tbody>
+			<tr>
+				<td><img alt="" src="https://via.placeholder.com/100" /></td>
+				<td>&nbsp;</td>
```

Comment:
Those `&nbsp;` makes me suspicious, especially that the related issue is about the extra space after the image ;)

Seems working properly without them, so please remove them from the content.